.buildkite/scripts/common/activate_service_account.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/vault_fns.sh
+
+ARG0="${1:-}"
+
+if [[ -z "$ARG0" ]]; then
+  echo "Usage: $0 <bucket_name|email>"
```

Comment:
I set up this script to make it easy to set up the impersonation before calling `gcloud storage / gsutil` commands. However, I didn't want the script caller to have to refer to e-mails that are associated with the service-accounts for the buckets.

Do you think it's okay to have this bucket name -> service account mapping (from line 49) to make it handier to use the script (with defaults/fallbacks for activating based on the email).